### PR TITLE
Add lazperf

### PR DESCRIPTION
**Pull request by chambbj**

### CODE REVIEW

File: recipes/lazperf/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+set -ex
+
+cmake -G "Unix Makefiles" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DWITH_TESTS=OFF
+
+make -j $CPU_COUNT
+make install
```
Comment (ocefpaf) [inline recipes/lazperf/build.sh]: Do we have a `make check`?

**Comment by chambbj** [comment]
@ocefpaf I've added `make test` as requested.

File: recipes/lazperf/bld.bat [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+      -DCMAKE_BUILD_TYPE:STRING=Release ^
+      -DWITH_TESTS:BOOL=OFF ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/lazperf/bld.bat]: Can we run the tests here too?

Comment (chambbj) [reply recipes/lazperf/bld.bat]: Sure, although Windows is disabled for now, but better to do it know before forgetting.

**Comment by chambbj** [comment]
@ocefpaf Windows tests enabled.

**Pull request merged**